var/npm/prepare-swagger-ui.js

```diff
@@ -0,0 +1,39 @@
+const fs = require('fs')
```

Comment:
I like to have the root clean, since usually in the modern web project there are already a lot of dot files out there.

So usually I have:
- `src` (for the code, which sometimes get compiled into the `dist`)
- `var` (for all the various files such as build scripts, images for the readme, etc.)
- maybe `__tests__`

And that's all.

Reply:
Since it is just one file you can leave it inside the root folder, it is not our common practice use the `var` folder and this could lead to confusion :)

Reply:
np, done